metadata_backend/api/handlers/user.py

```diff
@@ -124,12 +123,8 @@ async def delete_user(self, req: Request) -> Response:
 
         session.invalidate()
 
-        response = web.HTTPSeeOther(f"{aai_config['redirect']}/")
-        response.headers["Location"] = (
-            "/" if aai_config["redirect"] == aai_config["domain"] else f"{aai_config['redirect']}/"
-        )
-        LOG.debug("Logged out user ")
-        raise response
+        LOG.debug(f"Logged out user {user_id}")
+        return web.HTTPNoContent()
```

Comment:
add to list of changes, that logout no longer redirects user back, and frontend needs to handle it instead